deploy/environments/openshift-base/kustomization.yaml

```diff
@@ -22,12 +22,6 @@ configMapGenerator:
     disableNameSuffixHash: true
 
 # Include patches to update the Service, StatefulSet, Route, and RBAC resources.
-
-# Define the image to be updated.
-# images:
-# - name: ghcr.io/llm-d/placeholder
-#   newName: ghcr.io/llm-d/${IMAGE_TAG_BASE}
-#   newTag: ${VERSION}
```

Comment:
Please undo this change. The file as it was allows the YAML file to be used outside of the kind based tests.